README.md

```diff
@@ -42,7 +42,45 @@ jobs:
           token: ${{ secrets.BUF_TOKEN }}
 ```
 
-See [action.yml](action.yml) for all options.
+### Inputs
+
+| Parameter                       | Description                                        | Default            |
```

Comment:
It might be useful to have the type? We could also possible break these input down into sections.